src/Tools/AzureBlobStorage/AzureBlobStorage.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(ToolTargetFrameworks)</TargetFrameworks>
+    <RuntimeIdentifiers>linux-x64;linux-musl-x64;win-x64</RuntimeIdentifiers>
```

Comment:
`RuntimeIdentifiers` and `PackAsToolShimRuntimeIdentifiers` are not necessary unless you plan to publish or pack as a .NET CLI tool.